activiti-api-process-runtime/src/main/java/org/activiti/api/process/runtime/ProcessRuntime.java

```diff
@@ -123,5 +124,13 @@
     void removeVariables(RemoveProcessVariablesPayload removeProcessVariablesPayload); // review if we need to return removed variables// DO WE NEED THIS?>
 
     void setVariables(SetProcessVariablesPayload setProcessVariablesPayload); // review if we need to return set variables
+    
+    /**
+     * Get all subprocesses by pages using payload filters
+     */
+    Page<ProcessInstance> subprocesses(GetSubprocessesPayload getSubprocessesPayload,
```

Comment:
Same here, this extra method is not necessary you can reuse processInstances one.